packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -1,12 +1,72 @@
 import {color} from "../../tokens/color";
+import {mix} from "../../util";
+
+const core = {
+    trasparent: "transparent",
+    progressive: {
+        subtle: color.fadedBlue8,
+        default: color.blue,
+        strong: color.activeBlue,
+    },
+    destructive: {
+        subtle: color.fadedRed8,
+        default: color.red,
+        strong: color.activeRed,
+    },
+    neutral: {
+        subtle: color.fadedOffBlack8,
+        default: color.fadedOffBlack64,
+        strong: color.fadedOffBlack72,
+    },
+    disabled: {
+        subtle: "transparent",
+        default: color.fadedOffBlack32,
+        strong: color.fadedOffBlack50,
+    },
+    inverse: {
+        subtle: color.fadedOffBlack8,
+        default: color.offWhite,
+        strong: color.white,
+    },
+    // States
+    warning: {
+        subtle: color.fadedGold8,
+        default: color.gold,
+        strong: mix(color.offBlack32, color.gold),
+    },
+    success: {
+        subtle: color.fadedGreen8,
+        default: color.green,
+        strong: mix(color.offBlack32, color.green),
+    },
+};
+
+const background = {
```

Comment:
I'm taking a look at the new structure with the context of the Badge component!

If I'm understanding correctly, the Badge should use the semantic color status tokens. And those tokens use the core tokens! Is that right?

Should the status tokens use these background tokens instead? How do we decide when we should use the core tokens directly vs the abstraction?